core/apiCalls.js

```diff
@@ -66,11 +66,47 @@ export function fetchBlueprintInfoApi(blueprintName) {
 
 export function fetchModalCreateImageTypesApi() {
   const imageTypes = utils.apiFetch(constants.get_image_types)
-    .then(data => data.types)
+    .then(data => data.types.map(type => {
+      type.label = mapImageTypeLabels(type.name);
+      return type;
```

Comment:
Changing the input argument to of `map` often leads to confusions. It's not really a problem here, because `data` isn't used after calling `map`, but I prefer creating a new object:

```js
data.types.map(type => {
  return {
    name: type.name,
    label: imageTypeLabels[name] || name // see my suggestion below
  };
})
```